test/LibraryManager.Test/ManifestTest.cs

```diff
@@ -330,21 +332,18 @@ public void FromJson_Malformed()
             Assert.AreEqual("wwwroot", results.First().InstallationState.DestinationPath);
             Assert.AreEqual("jquery@3.2.1", results.First().InstallationState.LibraryId);
             Assert.AreEqual("cdnjs", results.First().InstallationState.ProviderId);
-            Assert.IsNotNull(results.First().InstallationState.Files);
+            Assert.IsNull(results.First().InstallationState.Files);
 
             // Valid parameters and files.
             var files = new List<string>() { "jquery.min.js" };
             results = await manifest.InstallLibraryAsync("jquery@2.2.0", "cdnjs", files, "wwwroot2", CancellationToken.None);
-            Assert.IsTrue(results.First().Success);
-            Assert.AreEqual("wwwroot2", results.First().InstallationState.DestinationPath);
-            Assert.AreEqual("jquery@2.2.0", results.First().InstallationState.LibraryId);
-            Assert.AreEqual("cdnjs", results.First().InstallationState.ProviderId);
-            Assert.AreEqual(1, results.First().InstallationState.Files.Count);
-            Assert.AreEqual("jquery.min.js", results.First().InstallationState.Files[0]);
+            Assert.IsFalse(results.First().Success);
+            Assert.AreEqual(1, results.First().Errors.Count);
+            Assert.AreEqual("LIB019", results.First().Errors[0].Code);
 
             // Valid parameters invalid files
             files.Add("abc.js");
-            results = await manifest.InstallLibraryAsync("jquery@3.3.1", "cdnjs", files, "wwwroot3", CancellationToken.None);
+            results = await manifest.InstallLibraryAsync("twitter-bootstrap@4.1.1", "cdnjs", files, "wwwroot3", CancellationToken.None);
```

Comment:
Changed the name of the library so we can proceed with test and not fail due to duplication of libraries names